package.json

```diff
@@ -194,6 +194,8 @@
     "tsx": "4.19.0",
     "typescript": "5.5.4",
     "unport": "0.6.0",
-    "vscode-languageclient": "9.0.1"
-  }
+    "vscode-languageclient": "9.0.1",
+    "compare-versions": "^6.1.1"
+  },
+  "packageManager": "pnpm@9.11.0+sha512.0a203ffaed5a3f63242cd064c8fb5892366c103e328079318f78062f24ea8c9d50bc6a47aa3567cabefd824d170e78fa2745ed1f16b132e16436146b7688f19b"
```

Comment:
is this expected?

Reply:
That's weird, corepack will set packageManager to package.json if there is no packageManager filed.